### PR TITLE
refactor(language-list): improve markup - INNO-1321

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Apply improvements discussed in [RFC](https://github.com/ec-europa/europa-component-library/issues/1003)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 15481e1adedc163438c1df9b7817debf1fe36503

https://deploy-preview-1007--europa-component-library.netlify.com

File: src/systems/ec/implementations/react/packages/language-list/LanguageListItem.jsx [hunk 1]
```diff
@@ -15,19 +15,26 @@ const LanguageListItem = ({ label, href, isActive }) => (
         size: 'xs',
       },
     }}
+    hrefLang={lang}
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/language-list/LanguageListItem.jsx]: `hrefLang` and `lang` attributes should be conditional, only appear when the `lang` parameter is a non-empty string

and if `isActive === false`? this is an open question. Is there any added value to adding these attributes if the link points to the current language?

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/implementations/react/packages/language-list/LanguageListItem.jsx (hunk 1 above)
Comment (emeryro) [reply src/systems/ec/implementations/react/packages/language-list/LanguageListItem.jsx]: I think it doesn't hurt to have it for active item too.
It is less usefull than for other items, but information is still relevant.

**Review by yhuard — CHANGES_REQUESTED** [review]
```js
    {...lang && {
      lang,
      hrefLang: lang,
      ...(isOverlay && { rel: 'alternate' }),
    }}
```
instead of 
```js
    {...lang && { hrefLang: lang }}
    {...lang && { lang }}
    {...isOverlay && { rel: 'alternate' }}
```
and then we're good :)

**Review by yhuard — CHANGES_REQUESTED** [review]
> Object.keys: argument is not an Object

Error in IE

**Review by yhuard — APPROVED** [review]

**Pull request merged**